jobconfig.toml

```diff
@@ -283,8 +285,8 @@
 [VIROLOGIST]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 1
-"Total Positions" = 1
+"Spawn Positions" = 2
+"Total Positions" = 2
```

Comment:
There's only enough machinery around for one virologist to comfortably work- keep this as-is.

Reply:
Change reverted in next commit.